### PR TITLE
Export Command

### LINKED ISSUE #13: As a user I can export the shown list to JSON
... so that I can share resources with others and generate insights from the data

### PR DESCRIPTION
**Pull request by cliftonfelix**
Resolves #13

- Added `ExportCommand` to export displayed list to a JSON file
- JSON file will be stored in `data/export/` folder
- Added tests for `ExportCommand`

### CODE REVIEW

File: src/main/java/seedu/address/storage/Storage.java [hunk 1]
```diff
@@ -29,4 +37,19 @@ public interface Storage extends AddressBookStorage, UserPrefsStorage {
     @Override
     void saveAddressBook(ReadOnlyAddressBook addressBook) throws IOException;
 
+    /**
+     * Saves the given {@link ObservableList} as a JSON file to the storage at filePathString.
+     * @param displayedList cannot be null.
+     * @param filePathString cannot be null.
+     * @throws IOException if there was any problem writing to the file.
+     */
+    static void saveDisplayedList(ObservableList<Person> displayedList, String filePathString) throws IOException {
```
Comment (cliftonfelix) [inline src/main/java/seedu/address/storage/Storage.java:46]: Is it okay to have a static method here?

Comment (lulucopter) [reply src/main/java/seedu/address/storage/Storage.java:46]: method name can be misleading as discussed

**Review by lulucopter — CHANGES_REQUESTED** [review]
Overall LGTM, do change the method names as discussed in the meeting

**Review by lulucopter — APPROVED** [review]
LGTM. Great job

File: src/main/java/seedu/address/storage/Storage.java [hunk 2]
```diff
@@ -29,4 +37,19 @@ public interface Storage extends AddressBookStorage, UserPrefsStorage {
     @Override
     void saveAddressBook(ReadOnlyAddressBook addressBook) throws IOException;
 
+    /**
+     * Exports the given {@link ObservableList} as a JSON file to the storage at filePathString.
+     * @param displayedList cannot be null.
+     * @param filePathString cannot be null.
+     * @throws IOException if there was any problem writing to the file.
+     */
+    static void exportDisplayedList(ObservableList<Person> displayedList, String filePathString) throws IOException {
```
Comment (maryjess) [inline src/main/java/seedu/address/storage/Storage.java:46]: I am not sure about this, but I feel it is better to move this static method to StorageManager instead? Not sure, what do the others think

Comment (cliftonfelix) [reply src/main/java/seedu/address/storage/Storage.java:46]: Will do this once Deon's PR is merged

**Pull request merged**